nipype/interfaces/ants/resampling.py

```diff
@@ -246,9 +246,13 @@ class ApplyTransformsInputSpec(ANTSCommandInputSpec):
     interpolation_parameters = traits.Either(traits.Tuple(traits.Int()),  # BSpline (order)
                                              traits.Tuple(traits.Float(),  # Gaussian/MultiLabel (sigma, alpha)
                                                           traits.Float())
-                                             )
-    transforms = InputMultiPath(
-        File(exists=True), argstr='%s', mandatory=True, desc='transform files: will be applied in reverse order. For example, the last specified transform will be applied first')
+                                            )
+    transforms = traits.Either(InputMultiPath(File(exists=True), desc='transform files: will be '
```

Comment:
I think this is too specific of fmriprep to be included in nipype.

I propose you to create in fmriprep a wrapper of this interface adding this new 'identity' option, instead of including this modification in nipype, that will affect a lot of users. WDYT?